xarray/tests/test_plot.py

```diff
@@ -267,6 +268,35 @@ def test2d_1d_2d_coordinates_contourf(self):
         a.plot.contourf(x="time", y="depth")
         a.plot.contourf(x="depth", y="time")
 
+    def test_contourf_cmap_set(self):
+        sz = (4, 4)
```

Comment:
```suggestion
        shape = (4, 4)
```
I find this easier to read